plugins/ui/src/deephaven/ui/components/spectrum/layout.py

```diff
@@ -101,3 +101,6 @@
 """
 A dictionary of CSS properties.
 """
+
+TextFieldLabelPosition = Literal["top", "side"]
+TextFieldLabelAlign = Literal["start", "end"]
```

Comment:
not specfic to text field, I think any component that takes a label prop also uses these ex. picker also accepts a label and label alignment/position

Reply:
Yes, just name them `LabelPosition` and `LabelAlign`:
```suggestion
LabelPosition = Literal["top", "side"]
LabelAlign = Literal["start", "end"]
```